packages/client/src/client.js

```diff
@@ -535,7 +535,11 @@ export class PercyClient {
     while (retries > 0 && !success);
 
     if (!success) {
-      this.log.error('Uploading comparison tile failed');
```

Comment:
delete

Reply:
throw error here

Reply:
Resolved.
- Now we are throwing an error here, which will be captured in snapshot.js under snapshotQueue queue(task), -> that will further get caught in error handler -> send for analysis and response back with the possible suggestion.